wordpress-mobile/WordPress-Android
# Remove unused UL&S code

Fixes #14274

This PR is a follow up to #14275 – which removed the `UNIFIED_LOGIN_AVAILABLE` flag from the app – and removes all the code that was not going to be used by the login library anymore. 

Note: I'm aware the changelog seems huge, but most of it consists of completely removed files and resources, so it _shouldn't_ be as hard to review as it looks.

The biggest change in terms of layout is that the base form layout now only has one button at the bottom (now called `bottom_button`) instead of two (previously called `primary_button` and `secondary_button`), as this change required successive changes to other layouts. 

Another significant change is that a few listener callbacks on `LoginActivity` are being completely removed (or just renamed, in case of `onTermsOfServiceClicked`). 

Other than that, it's also worth noting that all unused layouts, drawables and resources such as strings, dimensions and colors were either completely removed or moved out from the login library in cases where they were still being referenced outside the library. These changes were made in separate commits, so it's a bit easier to track them down.

Note: I'll prepare a new PR to push these changes back to the library once this is merged.

### To test

Run UI tests and make sure they all pass.

Smoke test the login flow and try out as many different flows as you feel is necessary and make sure they all are working as expected.

---
